### ISSUE TITLE
testing component: allow to run & test any model on separate test/validation set

### ISSUE THREAD

**Issue by martinjaggi** [epfml/disco#241]
for `csv` this seems already ok
https://epfml.github.io/disco/#/titanic/testing

but we might need this also for other modalities, to systematically test trained models on holdout data. this is because we don't want the train/test split to be affected by the number of nodes (1 or several). test set needs to remain constant.

alternative: allow an option in the task definition to **not** split the local data into train/test, but only provide as train
